modal/image.py

```diff
@@ -650,6 +650,27 @@ def my_build_function():
         """
         return self.extend(build_function=(raw_function, kwargs))
 
+    def env(self, **vars) -> "_Image":
```

Comment:
This seems a bit of an unusual interface to me. Wouldn't it be better to have this be a `dict[str, str]`? For example:

```python
modal.Image.conda().env({"CONDA_OVERRIDE_CUDA": "11.2"})
```

A dict is a more familiar interface to me because that's what [subprocess.run](https://docs.python.org/3/library/subprocess.html) uses.